### DISCUSSION TITLE
Mesh network and BLE

### DISCUSSION THREAD

**Discussion by mforbiz** [openthread/openthread#9266] [Q&A] [answered]
Hi.
I'm newbie to Thread.
Before to start a learning path, I'd like to know if Trhead may help me to do this.
I'd like to build a mesh network that can connect some ESP32 card (_say 10-20_) using BLE (_bluetooth low energy_) and NOT WiFi.
And if yes, may I use Arduino IDE to program ESP32 or I need other programming language?

**Comment by jwhui** [comment, marked as answer]
Thread is currently specified to operate over IEEE 802.15.4 radios - the same radio as Zigbee, but different than BLE.
